content/get-involved.ts

```diff
@@ -21,10 +28,11 @@ interface GetInvolvedContent {
   partnerPage: {
     header: string[];
     text: string[];
-    imgsrc: string;
+    imagesrc: string;
```

Comment:
A bit confusing that this one is `imagesrc` and on ln8 `imageSrc` can we align please

Reply:
I agree, what were we thinking? I have changed it.